igvm/src/snp_defs.rs

```diff
@@ -78,8 +83,14 @@ pub struct SevFeatures {
     pub debug_swap: bool,
     pub prevent_host_ibs: bool,
     pub snp_btb_isolation: bool,
-    #[bits(56)]
-    _unused: u64,
+    pub vmpl_supervisor_shadow_stack: bool,
+    pub secure_tsc: bool,
+    #[bits(4)]
+    _unused1: u8,
```

Comment:
nit: lets rename these two unused fields to reserved. I guess we didn't do that originally, but that matches better with the spec/common patterns.